inf-terraform-aws/files/variables.tf

```diff
@@ -0,0 +1,57 @@
+# -----------------------------------------------------------------------------
+# ENVIRONMENT VARIABLES
+# This stack supports the following secrets as environment variables.
+# -----------------------------------------------------------------------------
+
+# AWS_ACCESS_KEY_ID
+# AWS_SECRET_ACCESS_KEY
+# AWS_DEFAULT_REGION
+
+# -----------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# The following stack parameters require a value.
+# Documentation: https://www.terraform.io/docs/configuration/variables.html
+# -----------------------------------------------------------------------------
+
+# -----------------------------------------------------------------------------
+# OPTIONAL PARAMETERS
+# The following stack parameters are optional with sensible defaults.
+# Documentation: https://www.terraform.io/docs/configuration/variables.html
+# -----------------------------------------------------------------------------
+
+variable "name" {
+  description = "The name of the stack."
+  type        = string
+  default     = "stack-aws-quickstarter"
+}
+
+variable "meta_computer_system_name" {
```

Comment:
hm, outdated.

Reply:
we cant kick computer_system_name - shouldnt it be part of the TIR

Reply:
yes, but the TIR does not get it from this parameter. you will need it, however, to label the cloud resources